rllib/agents/qmix/model.py

```diff
@@ -20,6 +23,14 @@ def __init__(self, obs_space, action_space, num_outputs, model_config,
         self.fc1 = nn.Linear(self.obs_size, self.rnn_hidden_dim)
         self.rnn = nn.GRUCell(self.rnn_hidden_dim, self.rnn_hidden_dim)
         self.fc2 = nn.Linear(self.rnn_hidden_dim, num_outputs)
+        self.n_agents = model_config["n_agents"]
```

Comment:
Necessary definition to make QMIX work with the API.

Reply:
Is this because QMix is directly extending Policy instead of going through TorchPolicy (which sets up the state in/out stuff?